modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/MajorityPeerSelect.scala

```diff
@@ -0,0 +1,72 @@
+package org.tessellation.sdk.infrastructure.snapshot
+
+import cats.data.NonEmptyList
+import cats.effect.kernel.Async
+import cats.effect.std.Random
+import cats.syntax.all._
```

Comment:
if possible, please import exact syntax instead of all (as it impacts compilation time)

Reply:
Changed.